lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu'
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :order, :total, :menu
+
+  def initialize
+    @menu = Menu.new
+    # @menu.print_menu
+    @order = []
+  end
+
+  def add_items(item)
+    @order << @menu.menu[item - 1]
+  end
+
+  def check_order
+    @total = 0
+    @order.each do |item|
+      item.each do |item, price|
+        puts "#{item}: £#{price}"
+        @total += price
+      end
+    end
+    puts "Total: £#{@total}"
```

Comment:
This method is fairly long - it might be worth splitting this and putting the total calculation in a new method

Reply:
Great suggestion, you are right. It is quite long.